### ISSUE TITLE
Add BIP44 account addresses

### ISSUE THREAD

**Issue by cgcardona** [bigearth/bitbox-scaffold-vue#1]
Use `v-for` to loop over the list of BIP44 external change addresses and display them

https://vuejs.org/v2/guide/list.html

**Comment by zhaog100** [comment]
🙋 Claiming this bounty. I'll start work within 24-48h.

**Comment by 2bf** [comment]
Submitted PR #16 for this bounty: https://github.com/bigearth/bitbox-scaffold-vue/pull/16

It keeps the displayed mnemonic and the listed BIP44 external-change addresses derived from the same HD account, and renders each address with its derivation path using `v-for`.

Verification:
- `npm run lint`
- `npm run build` completed successfully; the legacy build process stayed open afterward under the current Node version, so I stopped the completed process after the build output was emitted.

If accepted for the BCH bounty, payment details can be coordinated privately.

**Comment by jdjioe5-cpu** [comment]
🙋 Reservation request for the $10 BCH bounty on this issue.

I see PR #16 by @2bf is already open and addresses the same BIP44 external-change address listing. I want to flag that transparently so the maintainer can pick the right next step.

A few quick questions before I do any implementation work:

1. Is the bounty still open and still paid in BCH on merge, or is PR #16 already considered the winning submission?
2. If the bounty is still open, may I be assigned this issue so I can submit an alternative or supplementary PR? I'm happy to work in parallel if multiple PRs are accepted, otherwise I'll wait.
3. If the bounty is going to PR #16, I will not start a competing implementation and will not open a second PR.

I have not opened a PR on this issue and I do not have a BCH receive address configured yet. I can share a public BCH (bitcoincash) address once the maintainer confirms the exact payout rail/network is still BCH; I will not claim a route is ready until then.

If no response is given in a few days, I'll treat the bounty as effectively closed and move on. No further action from me until I hear back.

**Comment by jdjioe5-cpu** [comment]
Re-affirming my prior reservation request on this $10 BCH bounty (issue #1) — see my earlier comment on 2026-08-11 (#issuecomment-5248167894) where I already flagged PR #16 by @2bf transparently.

Status remains:
- single authenticated claim (no duplicate request);
- implementation on hold pending receiver-route qualification;
- compatible BCH receive address can be provided immediately once the maintainer confirms the payout rail (mainnet/testnet/cashaddr format) — no route is being claimed as ready in the meantime.

If the maintainer is ready to proceed with PR #16, I'm happy to stand down; otherwise I'll wait for confirmation before any implementation work.
